doc/extending/extending_theano.txt

```diff
@@ -422,7 +505,8 @@ and ``b`` are equal.
 
         def make_node(self, x):
             # check that the theano version has support for __props__.
-            assert hasattr(self, '_props'), "Your version of theano is too old to support __props__."
+            assert hasattr(self, '_props'), "Your version of theano is too old 
+            to support __props__."
```

Comment:
I know this was in the original code, but I would remove it.  Theano has had support for __props__ for long enough.